lib/glific/flows/contact_action.ex

```diff
@@ -371,23 +375,31 @@ defmodule Glific.Flows.ContactAction do
 
     {_cid, message_vars} = resolve_cid(context, cid)
 
-    if is_nil(url) do
-      FlowContext.notification(context, "Could not send message to contact: Empty media URL")
-      {type, nil}
-    else
-      {:ok, message_media} =
-        %{
-          type: type,
-          url: url,
-          source_url: url,
-          thumbnail: url,
-          flow: :outbound,
-          caption: MessageVarParser.parse(caption, message_vars),
-          organization_id: context.organization_id
-        }
-        |> Messages.create_message_media()
-
-      {type, message_media.id}
+    db_url = from(m in MessageMedia, where: m.url == ^url) |> Repo.all()
```

Comment:
why Repo.all and not Repo.one ?

Reply:
Repo.one wasn't working throwing some error

Reply:
fixed it